.github/ISSUE_TEMPLATE/general-issue-template.md

```diff
@@ -12,12 +12,12 @@ assignees: carlyrevier, jehchow
 
 Welcome to the Microsoft Docs contributor guide feedback form!
```

Comment:
Microsoft Learn contributor guide

Reply:
At present time, the contributor guide title will stay the same.